api/http/workflow.go

```diff
@@ -214,12 +226,78 @@ func (h WorkflowController) StartWorkflow(c *gin.Context) {
 	}
 
 	c.JSON(http.StatusCreated, gin.H{
-		"id":   job.ID,
+		"id":   jobID,
 		"name": name,
 	})
 	l.Infof("StartWorkflow starting workflow %s : StatusCreated", name)
 }
 
+// StartBatchWorkflows responds to POST /api/workflow/:name/batch
+func (h WorkflowController) StartBatchWorkflows(c *gin.Context) {
```

Comment:
@alfrunes My intention was preprocessing the request to catch errors before starting the submission of the jobs, otherwise there is the risk of starting a subset of the workflows only. Shall we introduce a hard limit on the batch end-point, e.g. considering up to 1000 jobs per request?

Reply:
I still think that dispatching each request sequentially can be beneficial since we're in control of all parameters here. But let's not deal with this right now and leave it as a possible incremental improvement in case we see excessive memory usage in the future.

Reply:
OK, I agree, reworked to process the jobs sequentially.